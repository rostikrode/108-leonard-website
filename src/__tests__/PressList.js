// import 'babel-polyfill';
// import React from 'react';
// import ReactDOM from 'react-dom';
// import { mount } from 'enzyme';
// import PressList from '../components/pages/PressList';
// import pressJSON from '../components/data/press.json';
// import { MemoryRouter } from 'react-router-dom';
//
// window.gtag = window.gtag || function() {
//   return true;
// };
//
// it('renders without crashing', () => {
//   const tree = mount(
//     <MemoryRouter><PressList {...pressJSON} /></MemoryRouter>
//   );
//   expect(tree.html()).toMatchSnapshot();
// });
