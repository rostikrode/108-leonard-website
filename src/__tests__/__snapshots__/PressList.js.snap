// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div class=\\"press-wrapper \\"><ul class=\\"press-list \\"><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/new-york-times/luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">New York Times</h2><h5 class=\\"title sans\\">Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/wallpaper/100k-luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">Wallpaper</h2><h5 class=\\"title sans\\">$100K Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/the-real-deal/luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">The Real Deal</h2><h5 class=\\"title sans\\">Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/architects-newspaper/luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">Architects Newspaper</h2><h5 class=\\"title sans\\">Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/curbedcom/luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">Curbed.com</h2><h5 class=\\"title sans\\">Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li><li class=\\"press-item-wrapper\\"><div class=\\"press-item\\"><a aria-current=\\"false\\" href=\\"/press/forbes/luxury-conversion-downtown/\\"><h2 class=\\"publication sans-light-bold upper\\">Forbes</h2><h5 class=\\"title sans\\">Luxury Conversion Downtown</h5><h5 class=\\"date sans\\">9.10.17</h5></a></div></li></ul><img src=\\"down_arrow_large.svg\\" class=\\"arrow-down-scroll\\" alt=\\"downward arrow icon\\"></div>"`;
